sdk/tests/js/tests/tex-image-and-sub-image-utils.js

```diff
@@ -791,11 +791,71 @@ var TexImageUtils = (function() {
     return program;
   };
 
+  /**
+   * Return a list of unpack color spaces to test, supported by the specified
+   * WebGLRenderingContext.
+   */
+  var unpackColorSpacesToTest = function(gl)
+  {
+    if ('unpackColorSpace' in gl)
+      return ['srgb', 'display-p3'];
+    else
+      return [null];
```

Comment:
Would it be better to use `undefined` rather than `null` here? (Maybe not.)

Reply:
changed to undefined.